src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -143,8 +144,18 @@ public static void AssertValidValue(ISchema schema, IGraphType type, object inpu
 
             if (type is ScalarGraphType scalar)
             {
-                if (ValueFromScalar(scalar, input) == null)
-                    throw new InvalidValueException(fieldName, "Invalid Scalar value for input field.");
+                // validate value can be parsed correctly
```

Comment:
I do not understand this comment.

Reply:
hmmm.... well, what it does is attempt a conversion, testing to see if it converts properly, and discards the result

Reply:
how about `// verify value can be converted successfully` ?

Reply:
updated to the comment written above

Reply:
Do you mean "verif**iable** value can be converted successfully" ?

Reply:
no.  `validate` is a verb; verifiable is an adjective.  More correctly, I could write "This code validates that the value can be parsed".

Reply:
Actually I wrote `verify value can be converted successfully`.   Which could be written as "This code verifies that the value can be converted successfully."

Reply:
Now I see: verify **that** value can be ...
OK